app/models/spina/project.rb

```diff
@@ -13,5 +13,9 @@ class Project < ActiveRecord::Base
     accepts_nested_attributes_for :image_collection
 
     scope :newest_first, -> { order('completion_date DESC') }
+
+    def self.by_position
```

Comment:
I think a scope would be more appropriate for this ordering method i.e.

```
scope : by_position, -> { order('position ASC') }
```

Reply:
Is what I've done not considered a way of declaring scope? 

But sure, I'll switch it. Makes sense.

Reply:
A useful blog https://www.justinweiss.com/articles/should-you-use-scopes-or-class-methods/